show_help.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/python
+
+"""A helper to open the help documentation as a standalone dialog.
+
+Tim Sutton, 2016
+"""
+from safe.test.utilities import get_qgis_app, get_dock
+QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
+# from PyQt4.QtGui import QApplication
+from safe.gui.tools.help_dialog import HelpDialog
+from safe.gui.tools.help import definitions_help
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->